torchtext/csrc/sentencepiece.cpp

```diff
@@ -1,6 +1,23 @@
 #include <sentencepiece_trainer.h>
 #include <torch/script.h>
 
+#ifdef _WIN32
+#include <Python.h>
+#if PY_MAJOR_VERSION < 3
```

Comment:
nit: As torchtext dropped the Python 2 support, I feel this redundant. though it should not hurt.

Reply:
Fixed!